exporter/awscloudwatchlogsexporter/config.go

```diff
@@ -77,41 +83,14 @@ func (config *Config) Validate() error {
 	if config.QueueSettings.QueueSize < 1 {
 		return errors.New("'sending_queue.queue_size' must be 1 or greater")
 	}
-	if !isValidRetentionValue(config.LogRetention) {
-		return errors.New("invalid value for retention policy.  Please make sure to use the following values: 0 (Never Expire), 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 2192, 2557, 2922, 3288, or 3653")
+	if retErr := cwlogs.IsValidRetentionValue(config.LogRetention); retErr != nil {
```

Comment:
```suggestion
	if retErr := cwlogs.ValidateRetentionValue(config.LogRetention); retErr != nil {
```

I'd rename this function to align with `ValidateTagsInput()` and reflect that it is no longer returning a boolean as might be implied by the `Is...` prefix.

Reply:
I renamed the function in my latest commit.